guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -5,6 +5,20 @@ include::common/header.adoc[]
 
 = {UpgradingDocTitle}
 
+In this guide, the terms upgrade, update, and migrate have the following meanings:
+
+Upgrading::
+The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a y-stream release to the next, for example {Project} {ProjectVersionPrevious} to {Project} {ProjectVersion}.
+For more information, see xref:upgrading_process_overview[].
+Updating::
+The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a z-stream release to the next, for example {Project} {ProjectVersion}.0 to {Project} {ProjectVersion}.1.
+For more information, see xref:updating_project[].
+Migrating::
+The process of moving an existing {Project} installation to another hardware.
```

Comment:
Maybe "host" instead of hardware?

Reply:
Not a good idea, because it's about migrating the whole project, not a host

Reply:
Maybe "a new instance" instead of "another hardware"?